timemachine/fe/free_energy.py

```diff
@@ -1216,18 +1221,30 @@ def get_swap_acceptance_rates(fraction_accepted_by_pair):
             instantaneous_swap_acceptance_rates = get_swap_acceptance_rates(fraction_accepted_by_pair)
             average_swap_acceptance_rates = get_swap_acceptance_rates(np.sum(fraction_accepted_by_pair_by_iter, axis=0))
 
+            wall_time_per_frame_current = (current_time - last_update_time) / print_diagnostics_interval
+            wall_time_per_frame_average = (current_time - begin_loop_time) / (current_frame + 1)
+            estimated_wall_time_remaining = wall_time_per_frame_average * (md_params.n_frames - (current_frame + 1))
+
+            def format_wall_time(time_seconds):
+                return f"{time_seconds:.2f} s"
+
             def format_rate(r):
                 return f"{r * 100.0:5.1f}%"
 
             def format_rates(rs):
                 return " | ".join(format_rate(r) for r in rs)
 
-            print("Frame ", current_frame + 1)
-            print("Acceptance rates (inst.)   :", format_rates(instantaneous_swap_acceptance_rates))
-            print("Acceptance rates (average) :", format_rates(average_swap_acceptance_rates))
-            print("Final replica permutation  :", hrex.replica_idx_by_state)
+            print("Frame", current_frame + 1)
```

Comment:
Don't have a strong preference here; my main motivation for changing it was that "iteration" seemed vague, since there's no input parameter like `n_iterations`, whereas "frame" here ranges from 1 to `md_params.n_frames`.

Reply:
I was alright with it when there weren't timings on a 'per frame' basis, but now we are reporting times that aren't actually going to be per-frame, 1 frame for N windows.

Just seems like it could be an easy mistake for a user to make when interpreting the logs.